CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   speed improvement; it reads and writes faster than CSVs and produces smaller files (#200).
 - ``ModelFuture`` objects will emit any warnings which occurred during their
   corresponding CivisML job (#204)
+- ``ModelFuture`` objects will return an empty dictionary if validation metadata
```

Comment:
From what I can tell below `metrics` defaults to [`None`](https://github.com/civisanalytics/civis-python/pull/208/files#diff-f3b644e69a0d393e0ee202a864d7e190R557).

Reply:
I meant this line to refer to `validation_metadata`, not `metrics`, but it's obviously too vague. You raise a good point, though. It seems better to stay consistent here. I modified the code so that it would return `None` for both `validation_metadata` and `metrics` if the validation metadata were missing. This makes the behavior the same as it was before.

(Previously, skipping validation would store `None` in a file of validation metadata. In v2.1, we won't write the file at all.)